ThemingCodelab/build.gradle

```diff
@@ -1,11 +1,11 @@
 /*
- * Copyright 2020 The Android Open Source Project
+ * Copyright 2022 Google LLC
```

Comment:
Copyright is `The Android Open Source Project` not `google llc` - here and everywhere where you might have used this